tests/images.rs

```diff
@@ -180,21 +180,21 @@ fn generic_image() {
 
     let node = docker.run(generic_postgres);
 
-    let conn = Connection::connect(
-        format!(
-            "postgres://{}:{}@localhost:{}/{}",
-            user,
-            password,
-            node.get_host_port(5432).unwrap(),
-            db
-        ),
-        TlsMode::None,
-    )
-    .unwrap();
-    let rows = conn.query("SELECT 1+1 AS result;", &[]).unwrap();
-
-    assert_eq!(rows.len(), 1);
-    assert_eq!(rows.get(0).get::<_, i32>("result"), 2);
+    let connection_string = &format!(
+        "postgres://{}:{}@localhost:{}/{}",
+        user,
+        password,
+        node.get_host_port(5432).unwrap(),
+        db
+    );
+
+    let mut conn = Client::connect(connection_string, NoTls).unwrap();
+    let rows = conn.query("SELECT 1 + 1", &[]).unwrap();
+
+    for row in rows {
+        let first_column: i32 = row.get(0);
+        assert_eq!(first_column, 2);
+    }
```

Comment:
If for some reason, we would be returning 0 rows, the test would not have any assertions. We are only returning a single row so I don't think a loop is needed here :)

Reply:
ahh good catch, will change that back